src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -339,23 +333,81 @@ private void generateSummarySpreadsheet(
+          }
+          csvPrinter.print(nextRoundTally.subtract(thisRoundTally));
+        } else {
+          csvPrinter.print(0);
+        }
       }
       csvPrinter.println();
     }
 
-    csvPrinter.print("Inactive ballots");
+    csvPrinter.print("Active Ballots");
     for (int round = 1; round <= numRounds; round++) {
-      BigDecimal inactiveBallots = roundTallies.get(round).numInactiveBallots();
-
-      // The previous method of calculating inactive ballots was to sum values that included
-      // active ballots, leading to additional but unnecessary precision.
-      // Mimic that behavior here so there is no change in the test files.
-      BigDecimal activeBallots = roundTallies.get(round).numActiveBallots();
-      if (inactiveBallots.scale() < activeBallots.scale()) {
-        inactiveBallots = inactiveBallots.setScale(activeBallots.scale(), RoundingMode.HALF_UP);
+      csvPrinter.print(roundTallies.get(round).numActiveBallots());
+      csvPrinter.print("");
+      csvPrinter.print("");
+    }
+    csvPrinter.println();
+
+    csvPrinter.print("Current Round Threshold");
+    for (int round = 1; round <= numRounds; round++) {
+      csvPrinter.print(roundTallies.get(round).getWinningThreshold());
+      csvPrinter.print("");
+      csvPrinter.print("");
+    }
+    csvPrinter.println();
+
+    Pair<String, StatusForRound>[] statusesToPrint = new Pair[]{
+        new Pair<>("by Overvotes", StatusForRound.INACTIVE_BY_OVERVOTE),
+        new Pair<>("by Skipped Rankings", StatusForRound.INACTIVE_BY_SKIPPED_RANKING),
+        new Pair<>("by Exhausted Choices", StatusForRound.INACTIVE_BY_EXHAUSTED_CHOICES),
+        new Pair<>("by Repeated Rankings", StatusForRound.INACTIVE_BY_REPEATED_RANKING)
+    };
+
+    for (Pair<String, StatusForRound> statusToPrint : statusesToPrint) {
+      csvPrinter.print("Inactive Ballots by " + statusToPrint.getKey());
+      for (int round = 1; round <= numRounds; round++) {
+        StatusForRound status = statusToPrint.getValue();
+        BigDecimal thisRoundInactive = roundTallies.get(round).getBallotStatusTally(status);
+        csvPrinter.print(thisRoundInactive);
+        if (round != numRounds) {
+          BigDecimal nextRoundInactive = roundTallies.get(round + 1).getBallotStatusTally(status);
+          BigDecimal diff = nextRoundInactive.subtract(thisRoundInactive);
+          csvPrinter.print(diff);
+        } else {
+          csvPrinter.print(0);
+        }
+        csvPrinter.print("");
       }
+      csvPrinter.println();
+    }
+
+    csvPrinter.print("Inactive Ballots Total");
+    BigDecimal numUndervotes = roundTallies.get(1).getBallotStatusTally(
+        StatusForRound.INACTIVE_BY_UNDERVOTE);
+    for (int round = 1; round <= numRounds; round++) {
+      BigDecimal inactiveBallots = roundTallies.get(round).numInactiveBallots()
+          .subtract(numUndervotes);
 
       csvPrinter.print(inactiveBallots);
+      csvPrinter.print("");
+      csvPrinter.print("");
```

Comment:
add comment (here and below) clarifying why you do this (filling in unused cells)

Reply:
Done -- and actually, I should have been filling out the transfer value here like above, so that's done too.